src/dom/DomEvent.DoubleTap.js

```diff
@@ -44,6 +44,27 @@ export function addDoubleTapListener(obj, handler) {
 			return;
 		}
 
+		// When clicking on an <input>, the browser generates a click on its
+		// <label> (and vice versa) triggering two clicks in quick succession.
+		// This ignores clicks on elements which are a label with a 'ref'
+		// attribute (or children of such a label), but not children of
+		// a <input>.
+		if (e.composedPath) {
+			var path = e.composedPath();
+			if (path.some(function (el) {
+				return el instanceof HTMLLabelElement && el.attributes.ref;

```

Comment:
Is `ref` supposed to be [`for`](https://html.spec.whatwg.org/multipage/forms.html#attr-label-for)?

Reply:
...and that's why I should be fully awake before sending PRs.

Reply:
Then the following is wrong too?
> The <label> has a `ref` attribute (thus ignoring standalone labels and nested labels)